src/sentry/notifications/notifications/integration_nudge.py

```diff
@@ -83,6 +84,9 @@ def get_message_actions(
         return [
             MessageAction(
                 name="Turn on personal notifications",
+                label="Turn on personal notifications"
```

Comment:
note to self: check that this is the actual label of the current nudge notification